README.md

```diff
@@ -60,29 +60,6 @@ Balthasar Reuter (balthasar.reuter@ecmwf.int)
   move parameter structures to constant memory. To enable this variant,
   a suitable CUDA installation is required and the `--with-cuda` flag
   needs to be passed at the build stage.
-- **dwarf-cloudsc-gpu-scc-cuf-k-caching**: GPU-enabled and further
```

Comment:
I think this here was deleted by accident? Or was this intended?

Reply:
It'd also be great, if you could add a small section (or bullet point) to the README to tell users about the availability of the NEC variant. Probably a small paragraph?

Reply:
Sorry, the deleted part was not intentional.

I thought I already added how to compile on NEC in the README file. I'll check that again.